onnxscript/analysis_test.py

```diff
@@ -162,6 +175,16 @@ def nested():  # pylint: disable=unused-variable
 
         self.assertUses(f, {"x"})
 
+    def test_doc_string(self):
+        def f(x):
```

Comment:
I remember

```
@script()
def foo():

    @graph()
    def foo_graph():
        """Docstring"""
        ...
```

would cause an error. Does this fix handles it too?

Reply:
I hope so. At least, I tried an example like above, and it works.

Reply:
Possible to add a nested function def and see?

Reply:
So just

```
def foo():
    def foo_graph():
        """Docstring"""
        pass
```

Reply:
nested python function def I meant